pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -26,6 +26,7 @@ import (
 	kbv1 "github.com/elastic/cloud-on-k8s/v2/pkg/apis/kibana/v1"
 	"github.com/elastic/cloud-on-k8s/v2/pkg/utils/stringsutil"
 
+	octrl "github.com/tigera/operator/pkg/controller"
```

Comment:
I think most of these logstorage controller changes aren't right and should be reverted.